subgraph.yaml

```diff
@@ -6,9 +6,6 @@ schema:
 features:
   - nonFatalErrors
   - grafting
-graft:
```

Comment:
Why did we delete the graft though?

Reply:
we can comment it out, it's just the version I deployed was not a graft